glib-macros/src/lib.rs

```diff
@@ -443,20 +502,20 @@ pub fn genum_derive(input: TokenStream) -> TokenStream {
 /// use glib::prelude::*;
 /// use glib::subclass::prelude::*;
 ///
-/// #[derive(Debug, Copy, Clone, glib::GErrorDomain)]
-/// #[gerror_domain(name = "ExFoo")]
+/// #[derive(Debug, Copy, Clone, glib::ErrorDomain)]
+/// #[error_domain("ExFoo")]
```

Comment:
```suggestion
/// #[error_domain(name = "ExFoo")]
```

That's what @GuillaumeGomez wanted everywhere else so I guess this is just an oversight?

Reply:
Ah yes! Missed this one, good catch.